Doc/library/multiprocessing.rst

```diff
@@ -1936,11 +1941,21 @@ their parent process exits.  The manager classes are defined in the
 
       Create a shared :class:`list` object and return a proxy for it.
 
+   .. method:: set()
+               set(sequence)
+               set(mapping)
+
+      Create a shared :class:`set` object and return a proxy for it.
+
    .. versionchanged:: 3.6
       Shared objects are capable of being nested.  For example, a shared
       container object such as a shared list can contain other shared objects
       which will all be managed and synchronized by the :class:`SyncManager`.
 
+   .. versionchanged:: next
+      Add support for shared :class:`set`\s to :class:`multiprocessing.managers.SyncManager`
+      via :meth:`SyncManager.set() <multiprocessing.managers.SyncManager.set>`.
```

Comment:
By looking at the docs, I think it would be better to just have a `.. versionadded:: next` below `.. method:: set` and leave out "add support for shared sets [...]". The rationale is that we did this for Barrier but did not add any versioning for other objects.

Independently, it would still be good to have `.. versionadded:: next` below `.. method:: set` because that way the "versionadded" directive is tied to the `set()` method (the versionchanged:: 3.6 was actually for *all* shared objects, not just for dict/list).